tests/tools/fio/workload.go

```diff
@@ -4,29 +4,46 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
-	"strconv"
+
+	"github.com/pkg/errors"
 )
 
 // WriteFiles writes files to the directory specified by path, up to the
 // provided size and number of files
-func (fr *Runner) WriteFiles(path string, sizeB int64, numFiles int, opt Options) error {
-	fullPath := filepath.Join(fr.DataDir, path)
+func (fr *Runner) WriteFiles(relPath string, opt Options) error {
+	fullPath := filepath.Join(fr.LocalDataDir, relPath)
+	return fr.writeFiles(fullPath, opt)
+}
 
+func (fr *Runner) writeFiles(fullPath string, opt Options) error {
 	err := os.MkdirAll(fullPath, 0700)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "unable to make directory for write")
+	}
+
+	relWritePath, err := filepath.Rel(fr.LocalDataDir, fullPath)
+	if err != nil {
+		return errors.Wrapf(err, "error finding relative file path between %v and %v", fr.LocalDataDir, fullPath)
 	}
 
+	absWritePath := filepath.Join(fr.FioWriteBaseDir, relWritePath)
+
 	_, _, err = fr.RunConfigs(Config{
 		{
-			Name: fmt.Sprintf("write-%vB-%v", sizeB, numFiles),
-			Options: opt.Merge(Options{
-				"size":      strconv.Itoa(int(sizeB)),
-				"nrfiles":   strconv.Itoa(numFiles),
-				"directory": fullPath,
-			}),
+			Name: fmt.Sprintf("writeFiles"),
```

Comment:
```suggestion
			Name: "writeFiles",
```
Or do you want to preserve some identifying info in the name?